openstates/oh/bills.py

```diff
@@ -13,43 +13,333 @@
+                                bill.add_sponsor("cosponsor",sponsor_name)
+
+                            #actions
+                            blacklist = ["/solarapi/v1/general_assembly_131/resolutions/lr_131_0035-1/actions"]
+                            #the page in the blacklist just plain old failed to load
+                            #and everything got stuck
+                            if bill_version["action"][0]["link"] not in blacklist:
+                                action_doc = self.get(base_url+bill_version["action"][0]["link"])
+
+                                actions = action_doc.json()
+                                for action in actions["items"]:
+                                    actor = chamber_dict[action["chamber"]]
+                                    action_desc = action["description"]
+                                    try:
+                                        action_type = action_dict[action["actioncode"]]
+                                    except KeyError:
+                                        raise AssertionError("Unknown action {desc} with code {code}. Add it to the action_dict.".format(desc=action_desc, code=action["actioncode"]))
+                                        
+
+                                    date = datetime.datetime.strptime(action["datetime"],"%Y-%m-%dT%H:%M:%S")
+
+                                    committees = None
+                                    if "committee" in action:
+                                        committees = action["committee"]
+
+                                    bill.add_action(actor,
+                                                    action_desc,
+                                                    date,
+                                                    type = action_type,
+                                                    committees = committees)
+
+                            self.add_document(all_amendments, bill_id,"amendment",bill,first_page)
+                            self.add_document(all_fiscals,bill_id,"fiscal",bill,first_page)
+                            self.add_document(all_synopsis,bill_id,"synopsis",bill,first_page)
+                            self.add_document(all_analysis,bill_id,"analysis",bill,first_page)
+
+                            #todo: votes, committee votes
+                            vote_url = base_url+bill_version["votes"][0]["link"]
+                            vote_doc = self.get(vote_url)
+                            votes = vote_doc.json()
+                            self.process_vote(votes,vote_url,bill,legislators,chamber_dict,vote_results)
+
+                            vote_url = base_url+bill_version["cmtevotes"][0]["link"]
+                            vote_doc = self.get(vote_url)
+                            votes = vote_doc.json()
+                            self.process_vote(votes,vote_url,bill,legislators,chamber_dict,vote_results)
```

Comment:
I don't think there are many non-committee votes, but this might happen for floor votes as well (once they pop up in the LIS). That said, I haven't seen an instance of this happening yet.